product_simple_seasonality/security/seasonality.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Akretion
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.model.access" id="campaign_seasonality_access_name">
+        <field name="name">campaign.seasonality sale user</field>
+        <field name="model_id" ref="model_seasonality" />
+        <field name="group_id" ref="sales_team.group_sale_salesman" />
```

Comment:
I think any employee can read, but a really small subset of user can CUD because we don't always create season. Which existing group (settings one ?)